bcgov/critterbase-api
# Simsbiohub 472 containerize

## Changes
- New docker-compose.yml
- New dockerfile for api
- New Makefile for running commands
- Changed location of dockerfiles
- Added API logger

## Testing notes (Follow README)
- Run ```make env ```
- Run ```make backend ```
- All containers should complete successfully
- API accessible at http://localhost:9000/api
